azerothcore/azerothcore-wotlk
# fix(DB/GameObject-Spawn): Solid Chests respawn time in Azshara

Fix Respawn times for several Solid Chests located in Azshara.

_Please also see comments, pooling is added as well_

## Changes Proposed:
-  `spawntimesecs` increased from 180 to 7200 for chests with guids `48362, 48363,48364, 48365, 48366, 48367, 48368, 48369, 48370, 48371`


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4836

## Tests Performed:
- Rebuilt (docker), linux host. 
- Tested locally, verified that spawn time is not 180 anymore. Screenshot:
![Screenshot from 2021-03-13 16-18-39](https://user-images.githubusercontent.com/1468666/111044648-d229c880-8417-11eb-9972-c72aa066a58f.png)

## How to Test the Changes:
 - Teleport to any of `48362, 48363,48364, 48365, 48366, 48367, 48368, 48369, 48370, 48371` objects (use `.go gobject #object_guid`) 
 - Loot the chest.
 - Verify that the chest does not respawn in 3 minutes as it was before

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
